src/components/CheckboxInput.jsx

```diff
@@ -0,0 +1,22 @@
+import { useState } from "react";
+
+export function CheckboxInput(props) {
+  const { name, label } = props;
+  const [checked, setChecked] = useState(false);
+
+  const handleChange = () => {
+    setChecked((isChecked) => !isChecked);
+  };
+
+  return (
+    <label htmlFor={name}>
```

Comment:
voor mijn begrip'; die die name kan je dus voor verschillende checkboxes invullen en dus hergebruiken?

Reply:
Yes, is nu een herbruikbaar component!